pkg/find-interface_test.go

```diff
@@ -0,0 +1,19 @@
+package impl
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
I'd personally not use a testing framework, but that's ultimately up to @josharian :)